jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -66,7 +66,7 @@
 public class ArrayByteBufferPool implements ByteBufferPool, Dumpable
 {
     static final int DEFAULT_FACTOR = 4096;
-    static final int DEFAULT_MAX_CAPACITY_BY_FACTOR = 16;
+    static final int DEFAULT_MAX_CAPACITY_BY_FACTOR = 64;
```

Comment:
I don't like this change to the generic component (the buffer pool) driven by a component (zstd) that may not even be used.

I think it should be the other way around.